README.md

```diff
@@ -40,3 +40,19 @@ Currently the frontend is hosted in https://subvisuwheel.netlify.app/. To run lo
 ### Eslint, Prettier and Rubocop 
 
 `./bin/lint`
+
+### Active Admin customization
+
+This app allows for some customization, hosted in https://subvisuwheel.herokuapp.com/admin. You can create an admin account by running the console with `heroku run bin/rails c -a subvisuwheel` and then `AdminUser.create(email: "email_of_choice, password: password_of_choice")`.
```

Comment:
Should be: `AdminUser.create(email: "email_of_choice", password: "password_of_choice")`

Reply:
After changing this you can :shipit:

Reply:
Well noticed! Thanks